client/src/pages/organizations.js

```diff
@@ -6,72 +6,117 @@ import RepoCard from '../containers/repo-card';
 import UserCard from '../containers/user-card';
 import styled from '@emotion/styled';
 import { Alert } from '../components/alert';
+import TeamsCard from '../containers/teams-card'
+import MembersCard from '../containers/members-card'
 
-const GridContainer = styled.div(() => ({
-  display: 'flex',
-  flexDirection: 'row',
-  flexWrap: 'wrap',
-  alignSelf: 'center',
-  flexGrow: 1,
-  maxWidth: null,
-  width: '100%',
-}));
 
 /** TRACKS query to retrieve all tracks */
 const ORGS = gql`
-  query getUsers($key: String!) {
-    orgSearch(name: $key) {
+  query getOrgs($key: String!) {
+    orgsSearch(name: $key) {
       id
-      fullName
-      login
-      repos {
+      name
```

Comment:
This looks a bit messy, do you mind cleaning the spacing?

Reply:
I will, sorry